mlflow/store/artifact/s3_artifact_repo.py

```diff
@@ -3,33 +3,33 @@
 import posixpath
 from six.moves import urllib
 
-from mlflow import data
 from mlflow.entities import FileInfo
 from mlflow.exceptions import MlflowException
 from mlflow.store.artifact.artifact_repo import ArtifactRepository
 from mlflow.utils.file_utils import relative_path_to_artifact_path
 
 
+def parse_s3_uri(uri):
```

Comment:
Why add this code here and not use the parse_simple_uri from data.py ?

Reply:
Done